app/services/dttp/access_token_service.rb

```diff
@@ -0,0 +1,44 @@
+module Dttp
+  class AccessTokenService
+    class Client
+      include HTTParty
+      base_uri "https://login.microsoftonline.com"
```

Comment:
Should we put this in a constant?

Reply:
Will do. It gets set on boot so it's pretty much immutable anyway as it is. Not sure if there's another benefit of having it in a constant 🤷